decidim-meetings/db/migrate/20161130121354_create_meetings.rb

```diff
@@ -0,0 +1,18 @@
+class CreateMeetings < ActiveRecord::Migration[5.0]
+  def change
+    create_table :decidim_meetings_meetings do |t|
+      t.jsonb :title
+      t.jsonb :description
+      t.jsonb :short_description
+      t.datetime :start_date
+      t.datetime :end_date
+      t.text :address
+      t.jsonb :location
```

Comment:
`address` is the physical location of the place (221st B Baker Street), `location` is the name of the place (Sherlock Holmes' house)